prompt-management/features/essential/version-control.mdx

```diff
@@ -0,0 +1,157 @@
+
+## Managing Versions
+
+<Tabs>
+<Tab title="UI">
+
+Use the LangWatch UI to manage prompt versions:
+
+1. Navigate to the **Prompt Management** section
+2. Select a prompt to view its version history
+3. Click on any version to see the full configuration
+4. Use the version selector to switch between versions
+5. Create new versions by making changes and saving
+
+<Frame>
+  <img
+    className="block"
+    src="/images/prompts/prompt-versions-in-studio.png"
+    alt="LangWatch Prompt Versions in Studio"
+  />
+</Frame>
+
+</Tab>
+<Tab title="TypeScript SDK">
+
+LangWatch's TypeScript SDK supports retrieving prompts and specific versions:
+
+```typescript
+import { getPrompt, getPromptVersion, setupLangWatch } from "langwatch";
+
+// Initialize LangWatch for tracing and related features
+await setupLangWatch();
+
+// Get a specific prompt (latest version by default)
+const prompt = await getPrompt("customer-support-bot");
+
+// The prompt object contains version information
+console.log(`Prompt: ${prompt.name}`);
+console.log(`Version: ${prompt.version}`);
+console.log(`Version ID: ${prompt.versionId}`);
+
+// Get a specific version of a prompt
+const specificVersion = await getPromptVersion(
+  "customer-support-bot",
+  "version_abc123"
+);
+
+// Compile with variables
+const compiledPrompt = specificVersion.compile({
+  user_name: "John Doe",
+  user_email: "john.doe@example.com",
+  input: "I need help with my account",
+});
+```
+
+</Tab>
+<Tab title="REST API">
+
+Use the REST API to manage prompt versions:
+
+```bash
+# Get all versions of a prompt
+curl -X GET "https://app.langwatch.ai/api/prompts/{prompt_id}/versions" \
+  -H "Authorization: Bearer your-api-key"
+
+# Get a specific prompt (latest version)
+curl -X GET "https://app.langwatch.ai/api/prompts/{prompt_id}" \
+  -H "Authorization: Bearer your-api-key"
+
+# Create a new version
+curl -X POST "https://app.langwatch.ai/api/prompts/{prompt_id}/versions" \
+  -H "Authorization: Bearer your-api-key" \
+  -H "Content-Type: application/json" \
+  -d '{
+    "configData": {
+      "version": 2,
+      "prompt": "Updated prompt text...",
+      "messages": [...],
+      "inputs": [...],
+      "outputs": [...],
+      "model": "openai/gpt-4o-mini"
+    },
+    "commitMessage": "Improved customer support prompt"
+  }'
+```
+
+</Tab>
+</Tabs>
+
+## Version Information
+
+Each prompt version includes metadata such as version number, commit message, author, and creation timestamp. For detailed information about the version structure and data model, see the [Data Model](/prompt-management/data-model) page.
+
+## Version History
+
+LangWatch provides a clear view of how your prompts have evolved over time:
+
+<Frame>
+  <img
+    className="block"
+    src="/images/prompts/prompt-manager-edit.png"
+    alt="LangWatch Prompt Editor"
+  />
+</Frame>
```

Comment:
this screenshot is not really showing anything about version history, also it's the old UI

<img width="787" height="714" alt="image" src="https://github.com/user-attachments/assets/1741888b-8bb8-405f-9184-e0dce98aec3e" />